packages/framework/fluid-framework/src/index.ts

```diff
@@ -34,6 +34,27 @@ export type {
 	Myself,
 } from "@fluidframework/fluid-static";
 export type { SharedObjectKind } from "@fluidframework/shared-object-base";
+export type {
+	IErrorBase,
+	IEventProvider,
+	IDisposable,
+	IEvent,
+	IEventThisPlaceHolder,
+	IErrorEvent,
+	ErasedType,
+	IFluidHandle,
+	IFluidLoadable,
+	ITelemetryBaseProperties,
+	IEventTransformer,
+	IProvideFluidLoadable,
+	IFluidHandleErased,
+	TransformedEvent,
+	TelemetryBaseEventPropertyType,
+	Tagged,
```

Comment:
@markfields I wonder if this type name is overly general. Probably not for this PR, but I wonder if we should consider renaming this to be more clearly telemetry-specific?